recipe/meta.yaml

```diff
@@ -1,21 +1,25 @@
-{% set version = "2.7.0" %}
+{% set version = "2.8.0" %}
 
 package:
   name: tensorflow-split
   version: {{ version }}
 
 source:
   - url: https://github.com/tensorflow/tensorflow/archive/refs/tags/v{{ version }}.tar.gz
-    sha256: bb124905c7fdacd81e7c842b287c169bbf377d29c74c9dacc04f96c9793747bb
+    sha256: 66b953ae7fba61fd78969a2e24e350b26ec116cf2e6a7eb93d02c63939c6f9f7
     patches:
-      - patches/0001-relax-tolerence-in-special_math_test.patch
-      - patches/0002-skip-read-only-debugger_cli_common-test.patch
-      - patches/0003-relax-precision-in-matrix_log_op_test.patch
-      # requires setting GRPCIO_VERSION in build script
+      # commented out patches are unnecessary
```

Comment:
We should delete the patches if they are no longer needed.

Reply:
Yes, I just want to give the maintainers a chance to review first :)